packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-submission.jsx

```diff
@@ -37,7 +36,7 @@ const POISubmission = ({
             const is_onfido_supported =
                 selected_country.identity.services.onfido.is_country_supported && selected_country.value !== 'ng';
 
-            if (is_idv_supported && Number(idv_submissions_left) > 0 && !is_idv_disallowed) {
```

Comment:
I'm wondering why `idv_disallowed` flag is set from the API, better check for the root cause rather than removing the flag.

Reply:
Here we are getting 'idv_disallowed' from account status from API. these statuses are set considering the country of residence pointed during registration, and there are only few counties supported idv. 
But according to compliance team requirements proof of identity process shouldn’t be connected with country of residence, and if chosen during identity process country supports idv, then idv should be shown first

Reply:
@yauheni-kryzhyk-deriv  idv_disallowed is set not based on country or residence. It can be because of below reasons-

- status unwelcome
- landing company === svg
- client is age_verified
- allow_poi_resubmission
- manual status - expired or rejected
- onfido status - expired or rejected

So i suggest you to not remove the idv-disallowed check.

Reply:
Please get a confirmation from BE, as there is a similar issue for high risk clients, where the flag idv_disallowed is being set from BE

Reply:
thanks for your comments, team.
But this 'idv_disallowed' status is connected with current account settings or residence. and if you register new account you will have such status if the residence country is not in supported idv countries list.
But the compliance requirement is that if chosen during identity process country supports idv, idv should be shown